### PR TITLE
CI: Add `cuda-slow` test in FlexCI

### PR DESCRIPTION
**Pull request by kmaehashi**
Migrate slow tests from Jenkins to FlexCI.

### CODE REVIEW

**Comment by kmaehashi** [comment]
Testing: https://ci.preferred.jp/cupy.linux.cuda-slow/91281/

File: .pfnci/config.tags.json [hunk 1]
```diff
@@ -27,5 +27,6 @@
     "cupy.linux.cuda-rapids": ["full", "rapids"],
     "cupy.linux.cuda-example": ["full", "mini", "example"],
     "cupy.linux.cuda-head": ["head"],
+    "cupy.linux.cuda-head": ["slow"],
```
Comment (toslunar) [inline .pfnci/config.tags.json:30]:
```suggestion
    "cupy.linux.cuda-slow": ["slow"],
```

**Review by toslunar — CHANGES_REQUESTED** [review]

File: .pfnci/config.tags.json (hunk 1 above)
Comment (kmaehashi) [reply .pfnci/config.tags.json]: :+1::bow:

**Review by toslunar — APPROVED** [review]

**Pull request merged**